{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" { get }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/argumentparser\/validationerror\/message"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser\/ValidationError\/message","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The error message represented by this instance, this string is presented to"},{"type":"text","text":" "},{"type":"text","text":"the user when a "},{"type":"codeVoice","code":"ValidationError"},{"type":"text","text":" is thrown from either; "},{"type":"codeVoice","code":"run()"},{"type":"text","text":","},{"type":"text","text":" "},{"type":"codeVoice","code":"validate()"},{"type":"text","text":" or a transform closure."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"message","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:14ArgumentParser15ValidationErrorV7messageSSvp","modules":[{"name":"ArgumentParser"}]},"hierarchy":{"paths":[["doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser","doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser\/ValidationError"]]},"references":{"doc://org.swift.ArgumentParser/documentation/ArgumentParser/ValidationError/message":{"role":"symbol","title":"message","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"The error message represented by this instance, this string is presented to"},{"type":"text","text":" "},{"type":"text","text":"the user when a "},{"type":"codeVoice","code":"ValidationError"},{"type":"text","text":" is thrown from either; "},{"type":"codeVoice","code":"run()"},{"type":"text","text":","},{"type":"text","text":" "},{"type":"codeVoice","code":"validate()"},{"type":"text","text":" or a transform closure."}],"identifier":"doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser\/ValidationError\/message","kind":"symbol","type":"topic","url":"\/documentation\/argumentparser\/validationerror\/message"},"doc://org.swift.ArgumentParser/documentation/ArgumentParser":{"role":"collection","title":"ArgumentParser","abstract":[{"type":"text","text":"Straightforward, type-safe argument parsing for Swift."}],"identifier":"doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser","kind":"symbol","type":"topic","url":"\/documentation\/argumentparser"},"doc://org.swift.ArgumentParser/documentation/ArgumentParser/ValidationError":{"role":"symbol","title":"ValidationError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ValidationError"}],"abstract":[{"type":"text","text":"An error type that is presented to the user as an error with parsing their"},{"type":"text","text":" "},{"type":"text","text":"command-line input."}],"identifier":"doc:\/\/org.swift.ArgumentParser\/documentation\/ArgumentParser\/ValidationError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ValidationError"}],"url":"\/documentation\/argumentparser\/validationerror"}}}